docs/src/running-validator/vote-accounts.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Vote Account Management
+---
+
+This page describes how to set up an on-chain _vote account_.   Creating a vote
+account is needed if you plan to run a validator node on Solana.
+
+All settings for a vote account can be configured at the time the account is
+created, or they can be set after the account is created while the corresponding
+validator is running.
+
+### Vote Account Address
+
+A vote account is created at an address that is either the public key of a
+keypair file, or at a derived address based on a keypair file's public key and
+a seed string.
+
+The address of a vote account does not need to have a stored corresponding
+private key, because the address (although it may be a valid public key) is never
+needed to sign any transactions, but is just used to look up the account
+information.
+
+When someone wants to [delegate tokens in a stake account](../staking.md),
+the delegation command is pointed at the vote account address of the validator
+to whom the token-holder wants to delegate.
+
+### Validator Identity
+
+The _validator identity_ is a system account that is used to pay for all the
+vote transaction fees submitted by the vote account.
+Because the vote account is expected to vote on most valid blocks it receives,
+the validator identity account is frequently
+(potentially multiple times per second) signing transactions and
+paying fees.  For this reason the validator identity account's keypair must be
+stored as a "hot wallet" in a keypair file on the same system the validator
+process is running.
+
+Because a hot wallet is generally less secure than an offline or "cold" wallet,
+we only recommend storing enough SOL on the identity account to cover voting fees
+for some amount of time (perhaps a few weeks or months) and periodically topping
+off your validator identity account from a more secure wallet.
+
+The validator identity is required to be provided when a vote account is created.
+The validator identity can also be changed after an account is created by using
+the `vote-update-validator` command.
+
+### Vote Authority
+
+The _vote authority_ keypair is used to sign each vote transaction the validator
+node wants to submit to the cluster.  This doesn't necessarily have to be unique
+from the validator identity, as you will see later in this document.  Because
+the vote authority, like the validator identity, is signing transactions
+frequently, this also must be a hot keypair on the same file system as the
+validator process.
+
+It is recommended to set the vote authority to the same address as the validator
```

Comment:
Another recommendation, but a little worse, because it's in the passive voice. Who recommends?